dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -316,7 +356,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         xmlResult.reserve(c_maxAppNotificationPayload);
 
         // Build the button string and fill m_useButtonStyle
-        std::wstring buttons{ GetButtons() };
+        std::wstring actions{ GetActions() };
```

Comment:
Would be awesome to have a comment on how a final string would look like with the payload.

Reply:
//  <toast launch="action=AppNotificationClick;sequence={}">
    //    <visual>
    //      <binding template="ToastGeneric">
    //        <image placement="appLogoOverride" hint-crop="circle" src="image.png"/>
    //        <text>App Notification with Avatar Image< / text>
    //        <text>This is an example message using XML< / text>
    //      </binding>
    //    </visual>
    //    <actions>
    //      <input id="ComboBox" type="selection" defaultInput="yes">
    //        <selection id="yes" content="Going"/>
    //        <selection id="maybe" content="Maybe"/>
    //        <selection id="no" content="Decline"/>
    //      </input>
    //      <input id="textBox" type="text" placeHolderContent="reply"/>
    //      <action
    //        content="Send"
    //        imageUri="ms-appx://images/Icon.png"
    //        hint-inputId="textBox"
    //        arguments="action=reply&amp;threadId=92187"/>
    //      <action
    //        content="Open App"
    //        arguments="action=OpenApp&amp;sequence={}">
    //    </actions>
    //  </toast>

Reply:
I've tried to cobble something together with an image, some text, a combobox, a textbox and two buttons and I'm not convainced this would be useful.

I think i't better/easier to lookup the schema online when working in the builder.

I'm also concerned that this kind of comment will get out of sync quickly and become a maintenance nightmare.

I could create a unittest that creates an AppNotification simillar to this example comment. At least we know that this would never get out of sync with the code.

Reply:
Examples in the spec and documentation are welcome (and encouraged).

Not so useful as a comment.

Maybe useful as a doc-comment, if it can be small and meaningful.